pmrl.py

```diff
@@ -58,16 +58,16 @@ def is_wall(
 
 
 def draw_status(
+    root_console: libtcod.console.Console,
     status_y: Coordinates,
     player_hp: int
 ) -> None:
     msg = f'HP: {player_hp:2}'
-    libtcod.console_set_default_foreground(0, libtcod.blue)
```

Comment:
All references to setting fg color and printing are replaced with `root_console.print()`